src/_scss/pages/covid19/index.scss

```diff
@@ -1,384 +1,303 @@
+                h3.body__narrative {
+                    text-align: center;
+                    font-weight: $font-semibold;
+                    font-size: 3rem;
+                }
+
+                p.body__narrative-description {
+                    font-size: $small-font-size;
+                    text-align: center;
+                    width: 75%;
+                    margin: auto;
+                    padding-bottom: 5rem;
+                }
+
+                @import "../search/results/table/tableMessages";
+
+                .results-table-message-container {
+                    position: relative;
+                    background-color: rgb(246, 246, 246);
+
+                    .results-table-loading {
+                        margin-bottom: rem(100);
+                    }
+
+                    .no-results-container {
+                        text-align: center;
+                        margin-top: rem(100);
+                        margin-bottom: rem(100);
+
+                        .no-results-title {
+                            h4 {
+                                margin-bottom: 0;
+                                vertical-align: bottom;
+                            }
+                        }
+
+                        .no-results-message {
+                            p {
+                                margin-top: 0;
+                                vertical-align: top;
+                            }
+                        }
+                    }
+
+                    .loading-animation {
+                        @import "../search/results/screens/loading";
+                    }
+
+                    .results-table-error {
+                        .icon {
+                            width: rem(36);
+                            height: rem(36);
+
+                            svg {
+                                width: rem(36);
+                                height: rem(36);
+                            }
+                        }
+
+                        .title {
+                            font-size: rem(20);
+                            line-height: rem(45);
+                        }
+
+                        .description {
+                            font-size: rem(16);
+                            line-height: rem(10);
+                        }
+                    }
+                }
+            }
+
+            .body__section {
+                @import "../agency/v2/tabs";
+
+                @include display(flex);
+                @include flex-wrap(wrap);
+
+                margin-bottom: $global-mrg;
+
+                .body__header {
+                    width: 100%;
+                    display: flex;
+                    flex-wrap: wrap;
+                    align-items: center;
+                    justify-content: space-between;
+                    border-bottom: 3px solid $color-base;
+
+                    .body__header-left {
+                        display: flex;
```

Comment:
here too

Reply:
Yeah, realizing a bunch of these are from another PR but showing up because of formatting changes in `index.scss`. I can try to clean them up though.